xbmc/interfaces/json-rpc/schema/types.json

```diff
@@ -593,7 +593,8 @@
       "displayartist": { "type" : "string" },
       "votes": { "type": "integer" },
       "userrating": { "type": "integer" },
-      "sortartist": { "type" : "string" }
+      "sortartist": { "type" : "string" },
+      "isboxset": {"type" : "boolean" }
```

Comment:
This makes "isboxset" a property of songs too, I don't think that is right. 
Remove from here. It is already in Audio.Details.Album